doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md

```diff
@@ -0,0 +1,32 @@
+# Gas-cooled Microreactor Assembly Model Description
+
+The Gas-Cooled Microreactor (GC-MR) assembly model was developed at ANL as a modeling experiment that gathers several expected modeling challenges encountered by the U.S. industry. The GC-MR assembly uses a graphite structure, TRISO fuel blocks with 19.95 at% LEU fuel, YH2 moderator pins with FeCrAl envelope, and upper/lower BeO reflector regions. Additional modeling specificities were implemented in the GC-MR such as burnable poison blocks, helium coolant channels and a central control and shutdown rod.
+[Fig_1] illustrates the GC-MR assembly model. 
+
+!media media/gcmr/Fig_1.jpg
+      style=display: block;margin-left:auto;margin-right:auto;width:80%;
+      id=Fig_1
+      caption= Description of the GC-MR assembly model
+* Blocks contain both triangular and hexahedron elements*
```

Comment:
this is a little confusing because a libmesh block doesnt support 2 different element types

Reply:
They are two blocks according to @miaoyinb